scripts/test_nearlib.sh

```diff
@@ -16,19 +16,23 @@ git clone https://github.com/near/create-near-app.git
 # Make sure to use local nearcore for tests
 export NODE_ENV=local
 
+SRC_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export HOME=$SRC_DIR/../testdir
```

Comment:
it's a bit concerning we have to override `HOME`. Why? Shouldn't it be possible to pass NEAR home dir explicitly to  start_unittest.py (same as you can specify it explicitly with nearup)?

Reply:
both create-near-app and near-api-js (when NODE_ENV is local) assume validator_key.json is at `process.env.HOME/.near/validator_key.json` so this has to be overrided. It's fine to override home even when run locally, because it only in the sub-shell, won't affect HOME after script exit

Reply:
Maybe should fix in near-api-js and create-near-app side:
https://github.com/near/near-api-js/blob/master/test/config.js#L38
https://github.com/near/create-near-app/blob/master/blank_react_project/src/config.js#L44

But i think it's okay to override $HOME as it's only used in nearcore integration test, not worthy to not hard code in near-api-js/create-near-app

Reply:
@ailisp what is default path when I run local nearcore instance? Is it `process.env.HOME/.near/validator_key.json`? Or did it change? I think it did change for `nearup`, right?

Reply:
Yes nearup change it to be `HOME/.near/localnet/validator_key.json`, however the test here isn't using, maybe i should replace it using nearup localnet.

Reply:
Created separate issue to track local dev env fixes:
https://github.com/near/devx/issues/221